lua/open.lua

```diff
@@ -0,0 +1,12 @@
+local win = vis.win
+local results = {}
+results[1] = win.file.name == 'open.in'
+vis:open('open.aux')
+-- vis:command('e open.aux')
+results[2] = win.file.name == 'open.aux'
```

Comment:
`vis:open` currently creates a new window, as `:open` would do. However `win` will still refer to the old/previous window. If you want to access the new one use `vis.win` which should now point to the newly opened.